R/page.R

```diff
@@ -161,16 +252,59 @@ page_sidebar <- function(..., sidebar = NULL, title = NULL, fillable = TRUE, fil
 }
 
 
-#' @rdname page
-#' @inheritParams navset_bar
-#' @seealso [shiny::navbarPage()]
+#' Multi-page app with a top navigation bar
+#'
+#' @description
+#' `page_navbar()` provides a full-screen app with a navigation bar that behaves
+#' like a multi-page app (but is really a single-page app).
+#'
 #' @param fillable_mobile Whether or not `fillable` pages should fill the viewport's
 #'   height on mobile devices (i.e., narrow windows).
 #' @param underline Whether or not to add underline styling to page links when
 #'   active or focused.
 #' @param window_title the browser window title. The default value, `NA`, means
 #'   to use any character strings that appear in `title` (if none are found, the
 #'   host URL of the page is displayed by default).
+#' @inheritParams navset_bar
+#' @inheritParams page
+#'
+#' @seealso [shiny::navbarPage()]
+#'
+#' @examplesIf rlang::is_interactive()
+#' library(shiny)
+#' library(bslib)
+#'
+#' link_shiny <- tags$a(
+#'   shiny::icon("github"), "Shiny",
+#'   href = "https://github.com/rstudio/shiny",
+#'   target = "_blank"
+#' )
+#' link_posit <- tags$a(
+#'   shiny::icon("r-project"), "Posit",
+#'   href = "https://posit.co",
+#'   target = "_blank"
+#' )
+#'
+#' ui <- page_navbar(
+#'   title = "My App",
+#'   bg = "#0062cc",
+#'   underline = TRUE,
+#'   nav_panel(title = "One", p("First page content.")),
+#'   nav_panel(title = "Two", p("Second page content.")),
+#'   nav_spacer(),
+#'   nav_item(link_shiny),
+#'   nav_menu(
+#'     title = "Other links",
+#'     align = "right",
+#'     nav_panel("Three", p("Third page content.")),
+#'     nav_item(link_posit)
+#'   )
```

Comment:
I don't feel super strongly about this, but I guess we shouldn't necessarily be encouraging `nav_panel()` inside `nav_menu()` because of the accessibility issues (not like we'd ever not support that, though). 

```suggestion
#'   nav_menu(
#'     title = "Other links",
#'     align = "right",
#'     nav_item(link_shiny),
#'     nav_item(link_posit)
#'   )
```

Reply:
Opened a new issue to track the navset docs change.